include/crocoddyl/core/fwd.hpp

```diff
@@ -193,8 +195,10 @@ typedef ActivationModelWeightedQuadraticBarrierTpl<double> ActivationModelWeight
 typedef ActivationModelQuadTpl<double> ActivationModelQuad;
 typedef ActivationModelWeightedQuadTpl<double> ActivationModelWeightedQuad;
 typedef ActivationDataWeightedQuadTpl<double> ActivationDataWeightedQuad;
-typedef ActivationModelSmoothAbsTpl<double> ActivationModelSmoothAbs;
-typedef ActivationDataSmoothAbsTpl<double> ActivationDataSmoothAbs;
+DEPRECATED("Use ActivationModelSmooth1Norm", typedef ActivationModelSmooth1NormTpl<double> ActivationModelSmoothAbs;)
+DEPRECATED("Use ActivationDataSmooth1Norm", typedef ActivationDataSmooth1NormTpl<double> ActivationDataSmoothAbs;)
```

Comment:
Thanks for adding this here.

We also need to make `template <typename Scalar>ActivationModelSmoothAbsTpl` available as a deprecated class.

I think the way to do it that is still compatible with <C++11 standards would be to inherit from the new class

```
template <typename T>
class ActivationModelSmoothAbsTpl : public ActivationModelSmooth1NormTpl<T> { /* deprecated */};
```
and the same for the ActivationDataSmoothAbsTpl.

 Please do that in the file `core/activations/smooth-abs.hpp`, so that the header API isn't broken.

Reply:
done!
I also tested locally that the approach works.